### PR TITLE
MEN-3221 06.Server-installation/02.Demo-installation rewrite

### PR DESCRIPTION
**Pull request by merlin-northern**
ChangeLog:none
Signed-off-by: Peter Grzybowski <peter@northern.tech>

### CODE REVIEW

**Comment by merlin-northern** [comment]
guys I am so sorry, the previous one (https://github.com/mendersoftware/mender-docs/pull/985) is broken beyond recognition. I applied the changes here. sorry for the confuson.

File: 06.Server-installation/02.Demo-installation/docs.md [hunk 1]
```diff
@@ -5,7 +5,94 @@ taxonomy:
...
+```
+It will stop and remove the containers with all user data, but will not remove
+the images; you have to do it by hand.
+
+### Next steps
+
+You can now follow the help tooltips in the UI to guide you through each step
+of deploying to a device - accepting the device, viewing information about it,
+uploading an Artifact file, and finally deploying your very first update.
+Note, that the same process applies to [quickstart](../../get-started/connect-your-raspberry-pi)
```
Comment (estenberg) [inline 06.Server-installation/02.Demo-installation/docs.md:94]: Note that you can follow the [quickstart for hosted Mender](), just adjust the server address to your on-premise server.

Comment (merlin-northern) [reply 06.Server-installation/02.Demo-installation/docs.md]: sure

File: 06.Server-installation/02.Demo-installation/docs.md [hunk 2]
```diff
@@ -5,7 +5,94 @@ taxonomy:
...
 
-Migration note:
-- NOTE: Cover both Open source and on-premise Demo
-- NOTE: Shorten/simplify: Requirements, Install a Mender demo server, Install
-  the Mender client, Maintain the demo environment
+## Mender demo
+
+
+!!! The easiest way to test Mender is to use the [hosted Mender service](https://mender.io/signup).
+If you would rather like to run Mender on-premise, the Demo mode
```
[the remark this answers is not in the document]
Comment (merlin-northern) [reply 06.Server-installation/02.Demo-installation/docs.md]: sure

File: 06.Server-installation/02.Demo-installation/docs.md [hunk 3]
```diff
@@ -5,7 +5,94 @@ taxonomy:
...
+described here is the easiest way, but is not intended for production.
+The demo setup is not very different from the production one. In fact, they
+both run the same internals, and only differ slightly in the configuration. Namely,
+and since it is intended for testing only, the default keys are used, together with
```
[the remark this answers is not in the document]
Comment (merlin-northern) [reply 06.Server-installation/02.Demo-installation/docs.md]: sure removed fixed

**Pull request merged**